### DISCUSSION TITLE
Love this plugin! Just love it!

### DISCUSSION THREAD

**Discussion by ManasMadrecha** [nothingislost/obsidian-hover-editor#77] [General]
This is not an issue, so I am writing it in Discussions tab.

---

I just wanted to express my heartfelt thanks for creating such an amazing plugin. This is too good. ❤❤❤

Some subtle features like to be able to resize the popover and to be able to pin it anywhere on the screen are so useful. I didn't know I needed them, until I used them. This is like finding out tech features like regex and multi-cursors.

Obsidian in itself is already too good, and this plugin is not just a cherry on the cake; it is like a cake on the cake. 😂 
You have made my workflow so easy, it is beyond words. May you achieve all that you wish for. 
Stay blessed.

**Comment by nothingislost** [comment]
Thank you for the kind words! Glad to hear you're enjoying it :)

**Comment by huyz** [comment]
Yes this is a game-changer for me and I'm surprised that it's not better known.  Should even ship in the core editor.
